content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -172,20 +170,172 @@ Use `kubectl delete` to remove the Pod when you're finished:
 kubectl delete pod ephemeral-demo
 ```
 
-<!--
-Planned future sections include:
+## Debugging using a copy of the Pod
+
+Sometimes Pod configuration options make it difficult to troubleshoot in certain
+situations. For example, `kubectl exec` won't work with a container image that
```

Comment:
nit: [what situations?]: Sometimes Pod configuration options make it difficult to troubleshoot containers and applications running in containers?
nit: For example, you can't run `kubectl exec` to troubleshoot your container if your container image
does not include a shell or if your application crashes on startup.

Reply:
Applied the second suggestion, but I don't understand the first one. Is that the syntax for a reference link?

Reply:
Sorry @verb, I did not get a chance to look at the changes again before it was approved 😞

Reply:
Eh sorry... 😭